src/main/java/uk/gov/hmcts/ccd/domain/service/search/elasticsearch/security/ElasticsearchCaseSearchRequestSecurity.java

```diff
@@ -37,6 +40,9 @@ public CaseSearchRequest createSecuredSearchRequest(CaseSearchRequest caseSearch
     private String addFiltersToQuery(CaseSearchRequest caseSearchRequest) {
         BoolQueryBuilder boolQueryBuilder = QueryBuilders.boolQuery();
         boolQueryBuilder.must(QueryBuilders.wrapperQuery(caseSearchRequest.getQueryValue()));
+        BoolQueryBuilder grantTypeQueryBuilder = grantTypeESQueryBuilder
+            .createQuery(caseSearchRequest.getCaseTypeId());
+        boolQueryBuilder.must(grantTypeQueryBuilder);
 
```

Comment:
I thik we can delete ElasticsearchUserCaseAccessFilter, as we are now depending on the RoleAssignments instead of IDAM based authorisations and we don't need to check regular expression for the user logged in to know whether he has explicit access